tests/app/views/contexts/summary/test_answer.py

```diff
@@ -5,17 +5,18 @@
 
 @pytest.mark.usefixtures("app")
 @pytest.mark.parametrize(
-    "return_to",
-    ["section-summary", None],
+    "return_to, return_to_block_id",
+    [("section-summary", None), (None, None)],
 )
-def test_create_answer(return_to):
+def test_create_answer(return_to, return_to_block_id):
```

Comment:
The test asserts have not been updated, hence this is redundant since return_to_block_id is `None` in both cases. Need to test with a string value etc.